test/integration_test.rb

```diff
@@ -44,6 +44,31 @@ def test_tests_are_selected_based_on_changes_in_current_working_tree
         output = rake('ttnt:test:run')
         assert_match '2 runs, 2 assertions, 2 failures', output[:stdout]
       end
+
+      def test_storage_file_resides_with_rakefile
+        tmpdir = 'tmp'
+        Dir.mkdir(tmpdir)
+        git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt file')
+        File.delete("#{@repo.workdir}/.ttnt")
```

Comment:
`git_rm_and_commit` does not delete file in current working directory (unlike `git rm` command), so this line was needed.

However, to match the behavior of `git rm`, I made `git_rm_and_commit` delete the file in current working directory as well, and removed this `File.delete(...)` line. https://github.com/Genki-S/ttnt/commit/192fba92b4e3d9577d08715a6e19d8e6c343998c
Thank you for pointing out!